xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/store/XWikiHibernateBaseStore.java

```diff
@@ -274,12 +289,14 @@ public void updateSchema(XWikiContext inputxcontext, boolean force) throws Hiber
      * @param databaseProduct the database engine type.
      * @param inputxcontext the XWiki context.
      * @return the database/schema name.
+     * @deprecated use {@link HibernateAdapter#getDatabaseFromWikiName(String)} instead
      * @since 1.1.2
      * @since 1.2M2
      */
+    @Deprecated(since = "17.1.0RC1")
```

Comment:
Have we agreed on a strategy to eliminate the use of these obsolete methods?

Reply:
I would say the same as usual for deprecated methods: they should not be use anymore, and call to them removed when working on code which still manipulate them.